yt_dlp/extractor/nhk.py

```diff
@@ -67,12 +76,23 @@ def get_clean_field(key):
         }
         if is_video:
             vod_id = episode['vod_id']
+            api_url, api_token = self._get_vod_api(vod_id)
+            streams_json = self._download_json(api_url, vod_id, query={
+                'token': api_token,
+                'type': 'json',
+                'optional_id': vod_id,
+                'active_flg': 1,  # dont know what this is but site sends it
+            }, note='Downloading stream information')
+            stream_urls = traverse_obj(streams_json, ('meta', 0, 'movie_url'))
+            formats, subs = self._extract_m3u8_formats_and_subtitles(
+                stream_urls.get('mb_auto') or stream_urls.get('auto_sp') or stream_urls.get('auto_pc'),
+                vod_id)
+
             info.update({
-                '_type': 'url_transparent',
-                'ie_key': 'Piksel',
-                'url': 'https://movie-s.nhk.or.jp/v/refid/nhkworld/prefid/' + vod_id,
-                'id': vod_id,
```

Comment:
Should we still update the `id` here so that the video IDs stay consistent with what they were before the move away from piksel?

Reply:
oh, missed that
yes, that would be good